### PR TITLE
Use color variables instead of escape codes

### PR DESCRIPTION
**Pull request by casperklein**
# Description

This PR streamlines the color usage of the scripts. Color variables are now globally defined in `log.sh`. I did not use a separate file like `colors.sh`, because some scripts only source `log.sh`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [ ] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: target/scripts/helpers/log.sh
Comment (georglauterbach) [inline target/scripts/helpers/log.sh:25]: I was just thinking: maybe we should prefix this with `COLOR_`?

Comment (casperklein) [reply target/scripts/helpers/log.sh:25]: I thought about the same. But then decided to omit it:

- i think it's clear from the variable name what the usage is
- better readability in scripts with lots of colors (IMO), e.g. `${COLOR_LYELLOW}` is quiet long compared to `${YELLOW}`.

But I don't mind. We can change this. Lets wait what other maintainers say.

Comment (georglauterbach) [reply target/scripts/helpers/log.sh:25]: I'm absolutely fine with it as is, just wanted to ask here before we merge this and think about it later :)

**Review by polarathene — APPROVED** [review]
> i think it's clear from the variable name what the usage is

I'm not sure if that applies to `${RESET}` though?

Depending on which PR gets merged first, might want to keep https://github.com/docker-mailserver/docker-mailserver/pull/2535 in mind since it's adding a bunch of new `setup` commands that would need to be converted.

**Comment by casperklein** [comment]
> I'm not sure if that applies to ${RESET} though?

Standing alone, maybe not. But it's only used in context of color usage so I think it's fine.

**Review by georglauterbach — APPROVED** [review]
If we, in the future, realize that we may need the prefix we can always add it :)

**Pull request merged**
